### DISCUSSION TITLE
Feedback on the new UI and new Hide/show checkboxes

### DISCUSSION THREAD

**Discussion by StableInfo** [vladmandic/sdnext#2084] [General]
Hello, 
Sorry if this was discussed elsewhere, if yes, then feel free to redirect me to the right post, thanks.
I would like to say that I do not like the new UI sd.next made, mainly the hide/show checkboxes,

It's so confusing to use img2img tab, 

I am completely honest when I say that:
- I always spend multiple seconds searching for denoise bar or for the batch count bar.

I think the parameters should be brought back as all shown by default. If I want to use a simpler tool I can try to use the one that was made by the creator of ControlNet. No need to simpliify the UI.
I also dislike having them all vertically, maybe some of them should be pushed to the side (horizontally) a bit.

Thanks again for the hard work, I genuinly find the new UI is kinda problematic. 
Bythe way, I switched from a very very old version to the latest ones one month ago or so. Don't know when the UI was modified so much exaclty.
Thanks

**Comment by brknsoul** [comment]
Set up the UI the way you want, then go to Settings > User interface defaults. Click View Changes to make sure everything's the way you want, then hit Set New Defaults.

**Comment by StableInfo** [comment]
Where do I click? 
![image](https://github.com/vladmandic/automatic/assets/136593585/efdd2cbd-42e2-4e08-9a18-5dd6007ad419)

**Comment by StableInfo** [comment]
never mind just saw it.

**Comment by StableInfo** [comment]
But still, I still HATE that the checkbox "Second Pass" is SO FAR from what it hides/shows... its so NOT GENUINE.
Compare to the old hires fix button where if you click you see immedialtely the relevant sliders appearing direclty under the checkbox
Does that make sens what I am explaining?

**Reply by vladmandic** [reply]
I understand what you're saying. And we can agree to disagree. Main UI was very much overloaded for 95% of users. And everyone had different workflows, this is never going to be one-solution-fits-all.

If you have a suggestion how to improve it, I'm more than wiling to listen. Show all by default is not it.

**Reply by StableInfo** [reply]
I would like the "second pass" checkbox to show the hires sliders dieclty  under the checkbox and not several centimeters under,
![image](https://github.com/vladmandic/automatic/assets/136593585/eb35ed3c-bf82-44b7-8d42-906ee4809ef9)
I think this is the thing that bothered be most ^^
It does not feel genuine.
Maybe I am bias and USED to have extra sliders show up direclty under the button I am clicking instead of having sliders appearing somewhere under? I still think I might be unto something (for example imagine someone is very zoomed in, if he click this checkbox he might not even see what happened under).

**Reply by vladmandic** [reply]
Easily doable, but then each checkbox would need to be on a separate line and consume 4x screen area.
And please avoid "most", we're talking about a very subjective thing.

**Comment by StableInfo** [comment]
meant bothered "ME", i wrote "be" instead by mistake. Yes i meant it bothered me most (personal opinion).
I understand the problem of having extra lines for checkboxes though, I still dislike it.
- Feedback.
Anyone else feel free to add your feedback so vlad can have a better understanding of everyones opinion.
